upgrade/UPGRADE-v8.1.0-dev.md

```diff
@@ -285,6 +285,70 @@ There you can find links to upgrade notes for other versions too.
+            ```diff
+                 * @param \Shopsys\FrameworkBundle\Component\Error\ErrorPagesFacade $errorPagesFacade
+                 * @param \Shopsys\FrameworkBundle\Component\Domain\Domain $domain
+            +    * @param \Shopsys\FrameworkBundle\Component\Error\ErrorIdProvider $errorIdProvider
+                 */
+                public function __construct(
+                    ExceptionController $exceptionController,
+                    ExceptionListener $exceptionListener,
+                    ErrorPagesFacade $errorPagesFacade,
+                    Domain $domain
+                    Domain $domain,
+            +       ErrorIdProvider $errorIdProvider
+                ) {
+                    $this->exceptionController = $exceptionController;
+                    $this->exceptionListener = $exceptionListener;
+                    $this->errorPagesFacade = $errorPagesFacade;
+                    $this->domain = $domain;
+            +       $this->errorIdProvider = $errorIdProvider;
+                }
+            ```
+        - replace placeholder `{{ERROR_ID}}` by actual error ID in variable `$errorPageContent` in it
+            ```diff
+                private function createErrorPageResponse($statusCode)
+                {
+                    $errorPageStatusCode = $this->errorPagesFacade->getErrorPageStatusCodeByStatusCode($statusCode);
+                    $errorPageContent = $this->errorPagesFacade->getErrorPageContentByDomainIdAndStatusCode(
+                        $this->domain->getId(),
+                        $errorPageStatusCode
+                    );
+            +       $errorPageContent = str_replace('{{ERROR_ID}}', $this->errorIdProvider->getErrorId(), $errorPageContent);
+                    return new Response($errorPageContent, $errorPageStatusCode);
+                }
+            ```
+    - create new acceptance test in [`ErrorHandlingCest`](https://github.com/shopsys/shopsys/blob/master/project-base/tests/ShopBundle/Acceptance/acceptance/ErrorHandlingCest.php)
+        ```diff   
+        +    /**
+        +     * @param \Tests\ShopBundle\Test\Codeception\AcceptanceTester $me
+        +     */
+        +    public function test500ErrorPage(AcceptanceTester $me)
+        +    {
+        +        $me->wantTo('display 500 error and check error ID uniqueness');
+        +        $me->amOnPage('/test/error-handler/exception');
+        +        $me->see('Oops! Error occurred');
+        +        $cssIdentifier = ['css' => '#js-error-id'];
+        +        $errorIdFirstAccess = $me->grabTextFrom($cssIdentifier);
+        +        $me->amOnPage('/test/error-handler/exception');
+        +        $errorIdSecondAccess = $me->grabTextFrom($cssIdentifier);
+        +        Assert::assertNotSame($errorIdFirstAccess, $errorIdSecondAccess);
+        +    }
+        ```
```

Comment:
It's mandatory to regenerate error pages, right? Please state it in the upgrade instructions (even though they are regenerated during `build-deploy-part-2-db-dependent`). 
When testing it, I was confused for a moment before I realized it.